_data/compose-cli/docker_compose.yaml

```
@@ -265,7 +265,7 @@ options:
   description: |-
     DEPRECATED! USE --project-directory INSTEAD.
     Specify an alternate working directory
```

If callouts can be used in options' descriptions: 
```suggestion
> **Important**
>  This option is DEPRECATED. Use `--project-directory` to specify an alternate working directory.
```